source/common/upstream/priority_conn_pool_map_impl.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include "common/upstream/conn_pool_map_impl.h"
+#include "common/upstream/priority_conn_pool_map.h"
+
+namespace Envoy {
+namespace Upstream {
+
+template <typename KEY_TYPE, typename POOL_TYPE>
+PriorityConnPoolMap<KEY_TYPE, POOL_TYPE>::PriorityConnPoolMap(Envoy::Event::Dispatcher& dispatcher,
+                                                              const HostConstSharedPtr host) {
```

Comment:
I think we normally avoid const value parameters

Reply:
Not sure what I was going for here. :)